website/data/docs-nav-data.json

```diff
@@ -203,6 +203,10 @@
       {
         "title": "vault",
         "path": "configuration/vault"
+      },
+      {
+        "title": "ui",
+        "path": "configuration/ui"
```

Comment:
These items seem to be sorted alphabetically, so this item should be place before `vault`.

Reply:
🤦 I promise I know my ABCs song!

Reply:
Fixed in 57f045d